project/src/main/java/com/google/sps/utilities/DatabaseWrapper.java

```diff
@@ -123,4 +128,43 @@ public void insertVolunteeringOpportunity(VolunteeringOpportunity opportunity) {
     mutations.add(builder.build());
     return mutations;
   }
+
+  /**
+   * Given an eventId, retrieve all volunteering opportunities for that eventId
+   *
+   * @param eventId eventId for the event to retrieve volunteering opportunities for
+   * @return volunteering opportunties with given eventId
+   */
+  public Set<VolunteeringOpportunity> getVolunteeringOpportunitesForEventId(String eventId) {
+    SpannerOptions options = SpannerOptions.newBuilder().build();
+    Spanner spanner = options.getService();
+    DatabaseId db = DatabaseId.of(options.getProjectId(), instanceId, databaseId);
+    DatabaseClient dbClient = spanner.getDatabaseClient(db);
+
+    Set<VolunteeringOpportunity> results = new HashSet<VolunteeringOpportunity>();
+    try (ResultSet resultSet =
+        dbClient
+            .singleUse()
+            .executeQuery(
+                Statement.of(
+                    String.format(
+                        "SELECT VolunteeringOpportunityID, Name, NumSpotsLeft, RequiredSkills FROM"
+                            + " VolunteeringOpportunity WHERE EventID=\"%s\"",
+                        eventId)))) {
```

Comment:
Define a Statement statement = Statement.of
And .executeQuery(statement) here.

Reply:
Done.